log/experimental_level/level_test.go

```diff
@@ -75,10 +75,11 @@ func TestVariousLevels(t *testing.T) {
 
 func TestErrNotAllowed(t *testing.T) {
 	myError := errors.New("squelched!")
-	logger := level.New(log.NewNopLogger(), level.Config{
-		Allowed:       level.AllowWarnAndAbove(),
-		ErrNotAllowed: myError,
-	})
+	opts := []level.Option{
+		level.Allowed(level.AllowWarnAndAbove()),
+		level.ErrNotAllowed(myError),
+	}
+	logger := level.New(log.NewNopLogger(), opts...)
```

Comment:
No specific reason. I usually prefer to create a slice when I have multiple options because it looks more organized, especially if there's a chance the list of options will grow in the future. 

I'd be glad to change it if you have the opposite preference.